Yoast/wordpress-seo
# Fix alignment of the insert variable button in the Settings pages - Archives - special pages

## Summary

- Fixes the Insert variable button alignment in the Archive > Special pages > replacement variable editor

## Relevant technical choices:

- uses CSS flexbox as done for the other editors see https://github.com/Yoast/yoast-components/pull/618

## Test instructions

- switch yoast-components to the branch from https://github.com/Yoast/yoast-components/pull/618
- yarn ink to the plugin
- build JS
- verify under Archive > Special pages the replacement variable editor buttons are aligned correctly on both large screens and mobile screens; see desired layout in the screenshots below

<img width="717" alt="screen shot 2018-06-25 at 14 51 09" src="https://user-images.githubusercontent.com/1682452/41851202-6d31339e-7887-11e8-8c5b-d8dc9d5daf41.png">
<img width="429" alt="screen shot 2018-06-25 at 14 51 31" src="https://user-images.githubusercontent.com/1682452/41851203-6d4ee09c-7887-11e8-8d4b-0090ab0a6954.png">



Fixes #10138
